frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDetailsActions.tsx

```diff
@@ -78,6 +90,32 @@ const PipelineRunDetailsActions: React.FC<PipelineRunDetailsActionsProps> = ({ o
               ) : (
                 <React.Fragment key="compare-runs" />
               ),
+              !isRunActive ? (
+                <Tooltip
+                  position="left"
+                  content={
+                    <div>
+                      Archived runs cannot be restored until its associated experiment is restored.
```

Comment:
I might be missing something here, but if this messaging is not behind a feature flag, will it not be confusing to see this without the context of being within an experiment?

Reply:
I think its ok. Becuase the user will not be able to archive their experiment unless they have the feature flag flipped. So they would never get into the case. And if for some reason they do flip the flag, then archive, then flip back. We should still say why it is disabled